src/gitlab-ci.coffee

```diff
@@ -132,6 +135,47 @@ module.exports = (robot) ->
             return
 
         return "#{major}.#{minor}.#{dev}"
+    triggerBuildPost = (res, updateRichmond, quickly, branch, buildTagPrefix) ->
+        triggerURL = 'https://agra.sdelements.com/api/v4/projects/33/trigger/pipeline/'
```

Comment:
sure, normally you don't want to commit server names or URLs into a piece of code because when that server/URL changes you have to make a code change. In this case, we are also exposing the internals of our infrastructure to the public